{"uid":"892c823186ef585","name":"An uncaught error was detected outside of a test","fullName":"An uncaught error was detected outside of a test","historyId":"c59643030e1e776644fd5d1d6aa79cfc","time":{"start":1706768574351,"stop":1706768574482,"duration":131},"status":"failed","statusMessage":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","statusTrace":"TypeError: The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at ./cypress/support/e2e.js (webpack:///./cypress/support/e2e.js:20:10)\n    at __webpack_require__ (webpack:///webpack/bootstrap:19:0)\n    at 0 (http://localhost:37185/__cypress/tests?p=cypress/support/e2e.js:13777:18)\n    at __webpack_require__ (webpack:///webpack/bootstrap:19:0)\n    at eval (webpack:///webpack/bootstrap:83:0)\n    at eval (http://localhost:37185/__cypress/tests?p=cypress/support/e2e.js:87:10)\n    at eval (<anonymous>)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":4,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","statusTrace":"TypeError: The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at ./cypress/support/e2e.js (webpack:///./cypress/support/e2e.js:20:10)\n    at __webpack_require__ (webpack:///webpack/bootstrap:19:0)\n    at 0 (http://localhost:37185/__cypress/tests?p=cypress/support/e2e.js:13777:18)\n    at __webpack_require__ (webpack:///webpack/bootstrap:19:0)\n    at eval (webpack:///webpack/bootstrap:83:0)\n    at eval (http://localhost:37185/__cypress/tests?p=cypress/support/e2e.js:87:10)\n    at eval (<anonymous>)","steps":[],"attachments":[{"uid":"4a7a0d5a2d8d8404","name":"cypress/e2e/spec4.cy.js:2024-02-01T06:22:54.386Z","source":"4a7a0d5a2d8d8404.png","type":"image/png","size":38188},{"uid":"c783638242fc29a0","name":"video recording","source":"c783638242fc29a0.mp4","type":"video/mp4","size":39986}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"package","value":"cypress.e2e.spec4.cy.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"63ff07f181fc742f","status":"failed","statusDetails":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","time":{"start":1706768572335,"stop":1706768572447,"duration":112}},{"uid":"e9c6cd79112d89fd","status":"failed","statusDetails":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","time":{"start":1706768571279,"stop":1706768571478,"duration":199}},{"uid":"10b255af54d1291e","status":"failed","statusDetails":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","time":{"start":1706768570263,"stop":1706768570453,"duration":190}},{"uid":"b32c696ddecc35fa","status":"failed","statusDetails":"The following error originated from your test code, not from Cypress.\n\n  > cy.allure.writeEnvironmentInfo is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","time":{"start":1706768569435,"stop":1706768569621,"duration":186}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"892c823186ef585.json","parameterValues":[]}